src/vroom_write.cc

```diff
@@ -1,5 +1,6 @@
 #include "grisu3.h"
 #include <array>
+#include <functional>
```

Comment:
Seems fine to me. Do you have an opinion @DavisVaughan?

Reply:
Looks right to me since `cref()` does indeed come from `<functional>`
https://en.cppreference.com/w/cpp/utility/functional/ref